salt/scripts.py

```diff
@@ -46,9 +46,24 @@ def _handle_signals(client, signum, sigframe):
         hardcrash = client.options.hard_crash
     except (AttributeError, KeyError):
         hardcrash = False
+
+    exit_msg = '\nExiting gracefully on Ctrl-c'
```

Comment:
Hmm, that's curious. Debugging those are a pain. I'll cc: @kstreee since he was in that code not so long ago.